src/CollapsibleGroup/index.js

```diff
@@ -55,40 +64,39 @@ class CollapsibleGroup extends Component {
       oneIsOpen && `${baseClass}--is-open`,
     ].filter(Boolean).join(' ');
 
-    if (collapsibles && collapsibles.length > 0) {
+    const childrenAsArray = React.Children.toArray(children);
```

Comment:
We should destructure `Children` from React here, as we normally do for `Fragment`, and let's rename `childrenAsArray` to `arrayOfChildren`.